src/Talk/Delete/DeleteTalkAction.php

```diff
@@ -16,10 +17,15 @@
 use Symfony\Component\Security\Csrf\CsrfToken;
 use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
 
-final class Delete
+final class DeleteTalkAction implements Action
 {
+    /** @var TalkRepositoryInterface */
```

Comment:
Sorry, I guess I just hate useless comments 😄  These types can easily be inferred from the constructor type hint, no need to add extra maintenance.